src/Tasks/Backup/FileSelection.php

```diff
@@ -168,4 +168,20 @@ protected function sanitize($paths): Collection
                 return $path === false;
             });
     }
+
+    /**
+     * @param string $path
+     *
+     * @return array
+     */
+    protected function getMatchingPaths(string $path): array
+    {
+        if (strpos($path, '*') !== false && defined('GLOB_BRACE')) {
```

Comment:
What does this `if` statement do if you would explain it to a human? Refactor that if statement to a method of it's own with a good explanatory name.

Reply:
```php
protected function getMatchingPaths(string $path): array
{
     if (! defined('GLOB_BRACE')) {
         return glob($path);
     }
     return glob(str_replace('*', '{.[!.],}*', $path), GLOB_BRACE);
}
```
Better?? 
I'm sorry, maybe I have problems with my english

Reply:
I believe he wants you to split it into 2 functions:

```php
    /**
     * @param string $path
     *
     * @return array
     */
    protected function getMatchingPaths(string $path): array
    {
        if ($this->canUseGlobBrace($path)) {
            return glob(str_replace('*', '{.[!.],}*', $path), GLOB_BRACE);
        }

        return glob($path);
    }

    protected function canUseGlobBrace(string $path): bool {
         return strpos($path, '*') !== false && defined('GLOB_BRACE');
    }
```

Reply:
Thanks @sbrow

Reply:
Seems much better now thanks!

Is it possible to also add a test for this?

Reply:
I don't know how to remove constant `GLOB_BRACE` for a test